live/src/main/java/org/dbpedia/extraction/live/storage/DBpediaSQLQueries.scala

```diff
@@ -34,7 +34,7 @@ object DBpediaSQLQueries {
   }
 
   def getJSONCacheUnmodified: String = {
-    "SELECT pageID, updated FROM DBPEDIALIVE_CACHE WHERE updated <= now() - INTERVAL ? DAY ORDER BY updated ASC LIMIT ? "
+    "SELECT pageID, title, updated FROM DBPEDIALIVE_CACHE WHERE updated <= now() - INTERVAL ? DAY ORDER BY updated ASC LIMIT ? "
```

Comment:
The cache is initialized with pageID's only on the first run and some titles can be empty/null